.travis.yml

```diff
@@ -55,10 +54,14 @@ script:
   - stty cols $COLUMNS rows 40
   - tput cols; stty size
   - pushd inst
-  - octave --path=/home/travis/build/cbm755/octsympy/pytave -W --no-gui --eval "r = octsympy_tests; exit(r)"
+  - if [ "x$PYTAVE" = "xyes" ]; then
+        octave --path=$TRAVIS_BUILD_DIR/pytave -W --no-gui --eval "sympref ipc native; r = octsympy_tests; exit(r)";
+    else
+        octave -W --no-gui --eval "r = octsympy_tests; exit(r)";
+    fi
   - cat octsympy_tests.log
   - if [ "x$DOCTEST" = "xyes" ]; then
-        octave --path=/home/travis/build/cbm755/octsympy/pytave -W --no-gui --eval "pkg load doctest; syms x; r = doctest('.'); exit(~r)";
+        octave --path=$TRAVIS_BUILD_DIR/pytave -W --no-gui --eval "pkg load doctest; syms x; r = doctest('.'); exit(~r)";
```

Comment:
Since doctests are not run on daily builds so passing pytave directory doesn't really do anything but since it was already there so I didn't remove it.